spec/serializers/api/teacher_serializer_spec.rb

```diff
@@ -166,30 +181,6 @@
             expect(ect_enrolment["deferral"]).to be_nil
           end
 
-          context "when `training_status` is withdrawn" do
```

Comment:
Can we keep an assertion for the `training_status` and `participant_status` so we're expecting it to equal the `mock_training_status.status` and `mock_teacher_status.status` in both the ECF and mentor contexts for completeness/clarity?